src/spec/UtilSpec.ts

```diff
@@ -27,4 +27,22 @@ describe('Utility functions', () => {
       expect(ex.Util.clamp(12, -Infinity, 100)).toBe(12);
       expect(ex.Util.clamp(12, -100, Infinity)).toBe(12);
    });
-});
\ No newline at end of file
+
+   describe('removeItemFromArray function', () => {
+
+      it('should be a function', () => {
+         expect(typeof ex.Util.removeItemFromArray).toBe('function');
+      });
+
+      it('should return true when successfully removing an item', () => {
+         var arrayToRemove = ['Stark', 'Lannister', 'Targaryen'];
+         expect(ex.Util.removeItemFromArray('Lannister', arrayToRemove)).toBe(true);
+         expect(arrayToRemove.length).toBe(2);
+      });
+
+      it('should return false when item to delete is not present', () => {
+         var arrayToRemove = ['Godfrey', 'Crizzo', 'Fullstack'];
+         expect(ex.Util.removeItemFromArray('Lannister', arrayToRemove)).toBe(false);
+      });
+   })
```

Comment:
Ah, I am so sorry I missed that. Changing now...

Also, when running those tests I get this error on lines 34, 39, 45 `src/spec/UtilSpec.ts(45,25): error TS2339: Property 'removeItemFromArray' does not exist on type 'typeof "Util/Index"'.` Not sure what is breaking... 

Thanks for bearing with me today as I make my first PR, get used to a massive code base, and learn TypeScript/better git workflow :). It's been a lot of fun and I'd like to make more meaningful contributions as I'm certainly learning a lot about contributing very quickly